src/WarcraftLegacies.Source/Quests/Scourge/QuestDestroyStratholme.cs

```diff
@@ -21,28 +21,22 @@ public QuestDestroyStratholme(Capital stratholme, LegendaryHero arthas) : base("
       Required = true;
     }
 
-    private static bool IsPointValidForArthas(Point whichPoint) =>
-      !whichPoint.IsPathable(PATHING_TYPE_FLOATABILITY) && whichPoint.IsPathable(PATHING_TYPE_WALKABILITY);
-
     /// <inheritdoc />
-    protected override string RewardFlavour => "Prince Arthas could not protect the people of Stratholme. The Lich King's hold over him grows stronger.";
+    protected override string RewardFlavour => "Having failed to protect his people, Arthas seizes the cursed runeblade Frostmourne as the instrument of his vengeance. The malevolence of the blade overwhelms him. Arthas is now a loyal Death Knight of the Scourge, and will soon become its greatest champion.";
 
     /// <inheritdoc />
-    protected override string RewardDescription => "Gain the power Eye of the Lich King, which allows you to identify where Arthas is at any time";
+    protected override string RewardDescription => $"Learn to train {_arthas.Name} from the {GetObjectName(Constants.UNIT_UAOD_ALTAR_OF_DARKNESS)}";
```

Comment:
The quest doesn't actually give you Arthas... Have you tested this?
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/d55a7fa9-3987-4f7f-8178-0ec1fee1c4cb)

Reply:
i tested if it perma killed arthas, i didnt check if scourge gained him. thats my bad

Reply:
Sweet!